src/main/java/io/github/thebusybiscuit/slimefun4/utils/NumberUtils.java

```diff
@@ -171,6 +170,27 @@ private NumberUtils() {}
         }
     }
 
+    public static @Nonnull String getTimeLeftFromTick(int ticksLeft) {
+        /**
+         * This is the amount calculated in addProgress().
+         * This will be the scale used for timer.
+         */
```

Comment:
Is there a reason this is a block comment rather than just normal ones? Not a problem just a curiosity

Reply:
Honestly no clue I didn’t write it